server/lib/configuration.js

```diff
@@ -522,6 +522,14 @@ const conf = module.exports = convict({
     }
   },
   use_https: false,
+  userIpJwtSigning: {
```

Comment:
Hrm, yea I don't be believe we came to a consensus. I am indifferent to which but agree it should be consistent.